tcms/testcases/migrations/0007_convert_is_automated_to_boolean.py

```diff
@@ -16,6 +16,10 @@ def forward_migrate_data(apps, schema_editor):
         tc_history.save()
 
 
+def backward_empty_callable(apps, schema_editor):
+    pass
```

Comment:
The forwards migration converts from an int to bool, incl. historical records & the backwards migration should do the opposite. Consider True = 1, False = 0